channel_real_size: 2.4
channel_ratio: 0.7